.github/workflows/test.yml

```diff
@@ -189,6 +203,20 @@ jobs:
         with:
           api-key: ${{ env.RECORD_REPLAY_API_KEY }}
           public: true
+  end-e2e-marker:
```

Comment:
I think this job needs `if: always()` so it'll run even if the needed e2e tests fail.